test/e2e-tests.sh

```diff
@@ -85,6 +85,38 @@ $(dirname $0)/upload-test-images.sh e2e || fail_test "Error uploading test image
 # Setup resources common to all eventing tests
 setup_events_test_resources|| fail_test "Error setting up test resources"
 
-go_test_e2e ./test/e2e || fail_test
+go_test_e2e ./test/e2e
+exit_result=$?
+if [ ${exit_result} -ne 0 ]; then
+  # Collecting logs from all knative's eventing pods
+  echo "====== Collecting logs for namespace: knative-eventing ========"
+  for pod in $(kubectl get pod -n knative-eventing | grep Running | awk '{print $1}' ); do
+    echo "========== Collecting logs for pod: "${pod}" ================="
+    for container in $(kubectl get pod "${pod}" -n knative-eventing -ojsonpath='{.spec.containers[*].name}'); do
+      echo "----------------------------------------------------------"
+      echo "Container: "${container}
+      kubectl logs -n knative-eventing "${pod}" -c "${container}" || true
+      echo "----------------------------------------------------------"
```

Comment:
Include information about what is coming next, namely that it is previous runs of the container.

Reply:
I don't know what that means

Reply:
Right now it is printing a line of dashes. It should be something more like:

```shell
echo "----------------------------------------------------------"
echo "Previous run of Container: "${container}"
```